src/SourceBuild/content/eng/unifiedBuildValidation.targets

```
@@ -0,0 +1,19 @@
+<Project>
+
+  <PropertyGroup>
+    <UnifiedBuildValidationTestsDir>$([MSBuild]::NormalizeDirectory('$(RepoRoot)', 'test', 'Microsoft.DotNet.UnifiedBuild.Tests'))</UnifiedBuildValidationTestsDir>
+    <UnifiedBuildValidationTestsProject>$([MSBuild]::NormalizePath('$(UnifiedBuildValidationTestsDir)', 'Microsoft.DotNet.UnifiedBuild.Tests.csproj'))</UnifiedBuildValidationTestsProject>
+  </PropertyGroup>
+
+  <Target Name="RunUnifiedBuildValidation"
+          AfterTargets="Build"
+          DependsOnTargets="DetermineSourceBuiltSdkVersion">
```

You can now remove the DependsOnTargets:

```suggestion
          AfterTargets="Build">
```